org._2009scape.Launcher.json

```
@@ -0,0 +1,66 @@
+{
+    "id": "org._2009scape.Launcher",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "2009scape-launcher",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--filesystem=home/.local/share/2009scape",
```

```suggestion
        "--filesystem=xdg-data/2009scape",
```

Assuming you really need to access this.

---

Thanks :+1: